hasher-matcher-actioner/hmalib/lambdas/actions/action_evaluator.py

```diff
@@ -149,6 +152,19 @@ def get_threat_exchange_reaction_labels(
     return [ThreatExchangeReactionLabel("SAW_THIS_TOO")]
 
 
+def react_to_threat_exchange(
+    match_message: MatchMessage,
+    threat_exchange_reaction_label: ThreatExchangeReactionLabel,
+) -> None:
+    """
+    TODO implement
+    Puts a ThreatExchangeReactionMessage on the queue to be processed asynchronously
+    """
+    logger.info(
+        f"The contents of a ThreatExchangeReactionMessage will contain match_message = {match_message} and threat_exchange_reaction_label = {threat_exchange_reaction_label}"
```

Comment:
Nit: Split this line so it fits in ~800 characters, in py the easiest way is that adjacent strings are concatenated
```
  logger.info(
    "some line or another"
    "another line"
    f"an fstring {asdf}",  # comma goes here
  )

  # or
  logger.info(
    (
      "Another approach"
      "is to use wrapping parenthesis"
      "however, it's easy to accidentally add commas"
      "and turn it into a tuple"
    )
  )
```